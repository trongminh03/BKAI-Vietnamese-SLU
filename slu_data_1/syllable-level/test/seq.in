tăng giúp tớ bình nóng lạnh lên thêm tầm 46 độ c với nhé
