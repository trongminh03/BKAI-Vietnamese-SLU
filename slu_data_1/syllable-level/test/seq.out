B-command O O B-device I-device I-device O O O B-changing_value I-changing_value I-changing_value O O 
